### PR TITLE
Added configuration for firebase local Emulator suite.

### PR DESCRIPTION
**Pull request by Maahi10001**
## Description

To Configure Firebase Local Emulator suite for the codelabz.
We need to install the Local Emulator sutie in order to config it .The process to the I have already mention in #304.

## Related Issue

issue #234 

## Screenshots or GIF (In case of UI changes):
- Authentication on local emulator
image
- firestore on local emulator
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/config/index.js [hunk 1]
```diff
@@ -37,5 +37,4 @@ if (firebase.messaging.isSupported()) {
 }
 
 export const messaging = firebase_messaging;
-
```
Comment (Niweera) [inline src/config/index.js:40]: what is the change in this file?

Comment (Maahi10001) [reply src/config/index.js:40]: i guess i made a wrong pull i will make a new one

Comment (Maahi10001) [reply src/config/index.js]: I have updated the changes i think I missed it previously.

File: src/config/index.js [hunk 2]
```diff
@@ -19,9 +19,20 @@ const firebaseConfig = {
   measurementId: process.env.REACT_APP_FIREBASE_MEASUREMENTID,
 };
 
+console.log("firebaseConfig", firebaseConfig);
```
Comment (Niweera) [inline src/config/index.js:22]: do not put console log codes in commits, if you use console log for debugging remove them after.

Comment (Maahi10001) [reply src/config/index.js]: Yeah I used it for debugging, The changes updated.

**Review by Niweera — APPROVED** [review]

**Comment by Niweera** [comment]
Is the emulators export data when shutting down?

**Comment by Maahi10001** [comment]
> Is the emulators export data when shutting down?

am currently working on  import and export data from and to .json file I will be update it by this week.

**Review by Niweera — APPROVED** [review]

**Pull request merged**